PrebidMobile/PrebidMobile-core/src/main/java/org/prebid/mobile/AdUnit.java

```diff
@@ -109,7 +124,7 @@ public void fetchDemand(@NonNull final OnCompleteListener2 listener) {
         });
     }
 
-    public void fetchDemand(@NonNull Object adObj, @NonNull OnCompleteListener listener) {
+    public void fetchDemand(@Nullable Object adObj, @NonNull OnCompleteListener listener) {
```

Comment:
Why do we change the notation to Nullable? What is the use case? And, can we avoid it? 

It's a public API, and for the publishers, this method should always _demand_ an ad object. 

I see that it's due to migration to BidInfo. However, let's think about how we can avoid changing the public API.

Reply:
We use this method in the new multiformat API so it's possible that it's null. I removed this annotation and now old users will not think that nullable is okay.